test/data/css/customProperties.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <style>
+      .test-custom-props {
+          --prop1: val1;
+          --prop2:val2;
+          --prop3:val3 ;
+          --prop4: "val4" ;
+      }
+    </style>
+  </head>
+  <body>
+    <div class="test-custom-props"></div>
+    <script src="../../jquery.js"></script>
+    <script src="../iframeTest.js"></script>
+    <script>
+      startIframeTest( jQuery( ".test-custom-props" ) );
```

Comment:
Couldn't it be add to our test css?

> I agree it'd be better to do it that way, iframes are a little expensive so it's better to avoid them if they're not necessary.

Also, harder to support and debug those